src/highlight.js

```diff
@@ -238,6 +241,42 @@ https://highlightjs.org/
     }
   }
 
+  function commonKeyword(word) {
```

Comment:
In Ruby this is why was have `?`. :-)  Not 100% sold on is... is that a common JS idiom?

Reply:
I don't think there is a fixed naming scheme like in Java. JavaScript is more anarchic...
But I found this: https://medium.com/@amanhimself/javascript-conventions-74d6ff728082 (scroll down to "Function Naming"). 

> In JS, all we do is make functions in form of callbacks andevents. Author of Maintainable Javascript Nicholas Zakas introduces some common conventions for functions. I find this helpful:
> can Function that returns a boolean
> has Function that returns a boolean
> is Function that returns a boolean


And the google-styleguid also suggest "is" as prefix: https://google.github.io/styleguide/jsguide.html#naming-method-names

For me it's probably because I'm from the Java-world. But "commonKeyword" sounds like it *generates* a common keyword.

Reply:
Right, but CONTEXT is key... to me:

```js
if (commonKeyword("boy")) {
```

Is VERY readable in natural english...  the functions are never consumed in a vacuum. :)

Reply:
Maybe I read the `?` onto it in my mind, LOL.